server/src/core/server/models/site/index.ts

```diff
@@ -1,5 +1,5 @@
 import { identity, isNumber } from "lodash";
-import { MongoError } from "mongodb";
+import { Collection, MongoError } from "mongodb";
 import { v4 as uuid } from "uuid";
 
```

Comment:
I'm seeing a warning in the console: "(node:85676) [MONGODB DRIVER] Warning: cursor.count is deprecated and will be removed in the next major version, please use `collection.estimatedDocumentCount` or `collection.countDocuments` instead". 

Looks like we are using deprecated `count()` on line 135 in this file. Should we update to one of the suggested replacements now?

Reply:
looking into this, will see if I can update the query

Reply:
yup, was an easy fix, done!